<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>426</width>
    <height>270</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="inputMethodHints">
   <set>Qt::ImhDigitsOnly</set>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0" colspan="4">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="lineWidth">
      <number>5</number>
     </property>
     <property name="text">
      <string>Create P2P Path Segment</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Z:</string>
     </property>
    </widget>
   </item>
   <item row="7" column="3">
    <widget class="QPushButton" name="buttonSetStart">
     <property name="text">
      <string>set Startpoint</string>
     </property>
    </widget>
   </item>
   <item row="12" column="1">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>Z:</string>
     </property>
    </widget>
   </item>
   <item row="8" column="2">
    <widget class="QDoubleSpinBox" name="Box_Start_Z"/>
   </item>
   <item row="7" column="2">
    <widget class="QDoubleSpinBox" name="Box_Start_Y"/>
   </item>
   <item row="6" column="1">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>X:</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Y:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QDoubleSpinBox" name="Box_Start_X"/>
   </item>
   <item row="11" column="1">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Y:</string>
     </property>
    </widget>
   </item>
   <item row="10" column="1">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>X:</string>
     </property>
    </widget>
   </item>
   <item row="10" column="2">
    <widget class="QDoubleSpinBox" name="Box_End_X"/>
   </item>
   <item row="12" column="2">
    <widget class="QDoubleSpinBox" name="Box_End_Z"/>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Startpoint</string>
     </property>
    </widget>
   </item>
   <item row="11" column="2">
    <widget class="QDoubleSpinBox" name="Box_End_Y"/>
   </item>
   <item row="11" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Endpoint</string>
     </property>
    </widget>
   </item>
   <item row="11" column="3">
    <widget class="QPushButton" name="buttonSetEnd">
     <property name="text">
      <string>set Endpoint</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1" colspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>10</width>
       <height>30</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>10</width>
       <height>30</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>Box_Start_X</tabstop>
  <tabstop>Box_Start_Y</tabstop>
  <tabstop>Box_Start_Z</tabstop>
  <tabstop>Box_End_X</tabstop>
  <tabstop>Box_End_Y</tabstop>
  <tabstop>Box_End_Z</tabstop>
  <tabstop>buttonSetStart</tabstop>
  <tabstop>buttonSetEnd</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
